### PR TITLE
fixing noise problem

### PR DESCRIPTION
**Pull request by jinalklaulitz**
Summary of changes:
added noise variable and multiplied it to bad_img
changed conda install to pip install so it can be run at colaboratory
removed noise method calls at opencv comparison portion of the code

### CODE REVIEW

File: 10-autoencoder.ipynb [hunk 1]
```diff
@@ -231,26 +294,33 @@
...
+    "colab_type": "code",
+    "id": "H1oNdAxj2GrK",
+    "outputId": "6f30e1ae-a3d1-4fbb-9ee1-6eba91fce10b"
+   },
    "outputs": [],
    "source": [
     "# Compare the results: [noise], [img + noise], [img], [AE, Telea, Navier-Stokes] inpainting\n",
     "\n",
     "with torch.no_grad():\n",
-    "    display_images(img_bad.grad_fn.noise[3:7], img_bad[3:7])\n",
```
Comment (Atcold) [inline 10-autoencoder.ipynb]: This is the bug.
`img_bad.grad_fn.noise` does not exist.
Plz, revert the rest, and figure out where this tensor went.

File: 10-autoencoder.ipynb [hunk 2]
```diff
@@ -146,37 +177,53 @@
...
     "num_epochs = 1\n",
-    "# do = nn.Dropout()  # comment out for standard AE\n",
+    "do = nn.Dropout()  # comment out for standard AE\n",
     "for epoch in range(num_epochs):\n",
     "    for data in dataloader:\n",
     "        img, _ = data\n",
     "        img = img.to(device)\n",
     "        img.requires_grad_()\n",
     "        img = img.view(img.size(0), -1)\n",
-    "#         img_bad = do(img).to(device)  # comment out for standard AE\n",
```
Comment (Atcold) [inline 10-autoencoder.ipynb]: This is where the noise is introduced, by sending `img` to `do()` which is a dropout module.
